.test-infra/dataproc/init-actions/docker.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+#    This script installs and configures Docker on Google Cloud Dataproc Cluster.
+#    For information about which software components (and their version) are included
+#    in Cloud Dataproc clusters, see the Cloud Dataproc Image Version information:
+#    https://cloud.google.com/dataproc/concepts/dataproc-versions
+#
+#    This file originated from:
+#    https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/blob/master/docker/docker.sh
```

Comment:
Lets mention the git version from which we copied this file.
Otherwise we can just use the direct link to the dataproc init action in the create cluster script and download it automatically and use it to avoid code copy.

Reply:
ditto for flink init action.

Reply:
ok, mentioned in both files. The reason I copied the files is that I think it's better to be immune to any hypothetical changes in case scripts from the official buckets change and have the possibility to change the script easily (eg. configure flink/docker/beam as we need).